doc/shelley-genesis.md

```diff
@@ -503,16 +652,315 @@ You'll notice that the automagic method has divided the total supply amongst
+--TODO: What about creation of topology?
+--TODO: Probably need to add a default config.yaml for a shelley node
+--TODO: I needed to manuall update my time
+```
+$ cardano-node run \
+    --topology configuration/defaults/shelley/topology.json \
+    --shelley-kes-key example/KES/signkey \
+    --shelley-operational-certificate example/opcert \
+    --shelley-vrf-key example/VRF/signkey \
+    --config configuration/defaults/shelley/config.yaml \
+    --database-path example/db \
+    --port 3001 \
+    --socket-path example/socket/shelley-node
+```
+
+## Submitting a transaction
 
+Once you have a node up and running, you can submit transactions to it. You will need to:
+
+  1. Query the genesis UTxO
+  2. Create an address
+  3. Build a transaction
+  4. Sign the transaction
+  5. Submit the transaction
+
+#
+
+1. Querying the genesis UTxO. This will display the addresses and their funds that you defined in your genesis.json. It will allow you to get the transaction ids and their indexes to build the transaction.
+
+-- TODO: I had to create the socket file manually at this point and restart the node
+
+```
+cabal exec cardano-cli -- shelley query filtered-utxo \
+                        --address ${initialfundaddress} \
+                        --config configuration/defaults/shelley/config.yaml \
+                        --socket-path example/socket/shelley-node \
+                        --out-file /dev/null
+```
+You will get something like the following:
+
+```
+Filtered UTxO: UTxO (fromList [(TxIn (TxId {_TxId = 97681ef59a9d76c6db9247e7897d9e35c6f1c9d244ac233c0f30d02e2906549f}) 0,TxOut (Addr (KeyHashObj (KeyHash 97681ef59a9d76c6db9247e7897d9e35c6f1c9d244ac233c0f30d02e2906549f)) StakeRefNull) (Coin 1000000))])
+```
+
+
+2. You can send your ADA to another initial fund address or you can generate a new address to send ADA to.
+
+- Generate a keypair for the new output address
+
+```
+mkdir example/new-address
+cabal run cardano-cli:cardano-cli -- shelley address key-gen \
+                                   --verification-key-file example/new-address/verkey \
+                                   --signing-key-file example/new-address/signkey
+```
+
+- Build the address
+
+```
+cabal run cardano-cli:cardano-cli -- shelley address build \
+                                   --verification-key-file example/new-address/verkey
+```
+
+3. Build the transaction.
+- Be sure to add the correct index to the end of the txid using '#'.
+- Be sure to add the amount you like to spend at the end of the output using '+'.
+- Make sure the inputs = outputs + fee.
+
+```
+ cabal run cardano-cli:cardano-cli -- shelley transaction build-raw \
+                                    --tx-in 97681ef59a9d76c6db9247e7897d9e35c6f1c9d244ac233c0f30d02e2906549f#0 \
+                                    --tx-out 82065820debd78e08f0aa77ba6e30047f45e8335d908735f7dd73c808f56c27dc42a82a2+900000 \
```

Comment:
Can we have the example show to tx-outs? This confused me a bit at first, but if we sent the majority back to it's own address and a small amount to the newly generated address + fee I think it would clarify the usage better as seldom will someone want to drain an entire key and pay a fee doing so.

Reply:
for example, I use

```
cardano-cli shelley transaction build-raw --tx-in 6a362ab2f1f62d32367577f068bb90a45763f25722fd8320c44bf65a2aa68a39#1 --tx-out 820658206d30a984c860f31ed9cc15b553bb93888b520de2e2ef9016fe5086c73a284bf0+1000000000000 --tx-out 8206582025b36d57e010f92c3d6f806c790a56026e89c51ed1e6fa5e0c3e5ba353b1e0f1+9997999999800000 --ttl 49000 --fee 100000 --tx-body-file keys/wallet1/tx2.body
```